README.md

```diff
@@ -1 +1,38 @@
-# graasp-template-backend
+# graasp-plugin-validation
+
+# Purpose
+The Graasp Explorer is designed to be a platform to share and explore open educational resources. Inappropriate content, such as dirty words, aggressive langauge, hate speech and etc., should be prohibited from publishing.
+
+# Introduction
+## Validation Process
+There will be different validation processes to check different things. Each validation process will have a record in DB.
+
+### Existing
+- Bad Words Detection (JS)
+### In progress
+- Aggressive and Hate Speech Classification (Python)
+- Image Checking
+
+## Status
+To avoid any confusion, there are in total 5 distinct status in validation jobs.
+
+**Pending**: The process/review is still in progress.
+
+**Success**: The item is OK to publish.
+
+**Fail**: The item is suspicious, and could not be published at the moment. A failed process will wait for manual review from administrator.
+
+**Accept**: Admin accept the result of validation => The item is problematic and CANNOT be published.
+
+**Reject**: Admin reject the result of validation => The item is OK, and will be published.
+
+# Endpoints
+## Existing Ones:
```

Comment:
Remove this. Even if there's endpoints not existing I would just add `(to be added)` or similar next to the endpoint definition.